clients/src/test/java/org/apache/kafka/common/protocol/types/ProtocolSerializationTest.java

```diff
@@ -87,6 +89,15 @@ public void testSimple() {
         check(Type.INT16, (short) -11111, "INT16");
         check(Type.INT32, -11111111, "INT32");
         check(Type.INT64, -11111111111L, "INT64");
+        check(Type.FLOAT64, -0.5D, "FLOAT64");
+        check(Type.FLOAT64, 1e300D, "FLOAT64");
+        check(Type.FLOAT64, 0.0D, "FLOAT64");
+        check(Type.FLOAT64, -0.0D, "FLOAT64");
+        check(Type.FLOAT64, Double.MAX_VALUE, "FLOAT64");
+        check(Type.FLOAT64, Double.MIN_VALUE, "FLOAT64");
+        check(Type.FLOAT64, Double.NaN, "FLOAT64");
+        check(Type.FLOAT64, Double.NEGATIVE_INFINITY, "FLOAT64");
+        check(Type.FLOAT64, Double.POSITIVE_INFINITY, "FLOAT64");
```

Comment:
Do we want to have a simple positive value like `2.5`?

Reply:
Done.